src/cmd/cmd_conc.ml

```diff
@@ -0,0 +1,240 @@
+open Syntax
+module Expr = Smtml.Expr
+module Value = Concolic_value.V
+module Choice = Concolic.P.Choice
+
+let () = Random.self_init ()
```

Comment:
We should maybe store this in some kind of cache file, in order to be able to get some deterministic results when needed?

Reply:
Yeah, probably at some point we could use a default seed, in my experience it makes debugging easier. 

For now, since it only runs 1 path its good because we can see different paths with different executions :smiley:

Reply:
Yes I fixed the default seed. That was mainly to be able to test before having some proper runner